src/main/resources/application.yml

```
@@ -1,3 +1,3 @@
 spring:
-  application:
-    name: WellMeet-Backend
+  profiles:
+    active: local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)